apps/fxlab/app/src/main/kotlin/com/mobileer/androidfxlab/MainActivity.kt

```diff
@@ -154,7 +154,7 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun addDefaultEffects() {
-        val defaultEffects = listOf("Overdrive", "White Chorus", "Echo")
```

Comment:
Sounds more interesting? Curious about the inclusion of gain as a default since it is quite "boring"